### DISCUSSION TITLE
Regional Pricing github copilot

### DISCUSSION THREAD

**Discussion by gerardosabetta** [community/community#24594] [Copilot Conversations]
<p>Hey Github support. I’m located in Argentina. A lot of digital services offer an adjusted price for Argentina and other countries with low purchasing power. It’s called . <a href="https://en.wikipedia.org/wiki/Purchasing_power_parity" rel="noopener nofollow ugc">Purchasing power parity</a>. I wanted to know if it would be possible to access this kind of discount.</p>

**Comment by queenofcorgis** [comment]
<p>Hi there :wave:t3:</p>
<p>I reached out to the Copilot team to get an answer on this, please stay tuned!</p>

**Comment by gerardosabetta** [comment]
<p>Hey <a class="mention" href="/liligalante">@liligalante</a> any updates on this?</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p><a class="mention" href="/gerardosabetta">@gerardosabetta</a> thanks for your patience. Unfortunately this is not a discount users will have access to, but you are able to get a discount by purchasing annually instead of monthly.</p>

**Comment by ghost** [comment]
<p>this is very disappointing, it is obvious that no one in South America will pay that price, it is too expensive. Thanks for looking an answer anyway</p>

**Comment by Coystark** [comment]
$10/m for all SA countries is very expensive, even in BR the Dollar is 1 for 5 BRL.
Very disappointing.

**Comment by Wep-wep** [comment]
Creo que al usar una PC cabe la posibilidad de que pueda pagar dicha cantidad

**Comment by manikanta-hitunik-com** [comment]
For Indian region too its too high. and for certain Asian countries.

**Comment by gkonuralp** [comment]
Yearly price $100 is:
7,96% of US monthly minimum wage. 
28.21% of Turkey monthly minimum wage.

**Reply by yLordMagnus** [reply]
For brazil it is 44% of the monthly minimum wage.
$100,00 -> R$534,00 -> Minimum wage is R$1.212,00 -> 44%

**Reply by amr3k** [reply]
In Egypt, that would be 100% of official monthly minimum wage, a lot of businesses don't even respect that law, and I know a lot of people work for less than what equals $50/month.

GitHub should really rethink their pricing.

**Reply by DevWael** [reply]
> In Egypt, that would be 100% of official monthly minimum wage, a lot of businesses don't even respect that law, and I know a lot of people work for less than what equals $50/month.
> 
> GitHub should really rethink their pricing.

It's now 1 USD equals 31 EGP

**Reply by AmrMonier** [reply]
> > In Egypt, that would be 100% of official monthly minimum wage, a lot of businesses don't even respect that law, and I know a lot of people work for less than what equals $50/month.
> > GitHub should really rethink their pricing.
> 
> It's now 1 USD equals 31 EGP

it's almost 50EGP now

**Reply by DevWael** [reply]
Update:
It's now 1 USD equals 48 EGP

**Comment by saurabhsri108** [comment]
Get the hint from their reply, it's not for the poor countries. 

PS: Speaking as an Indian myself from India who hasn't purchased it because pricing.

**Reply by gustavopch** [reply]
People like to talk about accessibility, but making products accessible to people in poor countries... nah, who cares.

**Comment by luchux** [comment]
Extremely expensive! one more time a company that increases the GAP, more power for the children of the commonwealth - india. So disappointed.
